lib/telegram/bot/types/keyboard_button_request_users.rb

```diff
@@ -3,10 +3,11 @@
 module Telegram
   module Bot
     module Types
-      class KeyboardButtonRequestUser < Base
+      class KeyboardButtonRequestUsers < Base
         attribute :request_id, Types::Integer
         attribute? :user_is_bot, Types::Bool
         attribute? :user_is_premium, Types::Bool
+        attribute? :max_quantity, Types::Integer
```

Comment:
I'd add the default and probably even range validation:

> 1-10. Defaults to 1.

Reply:
Yep, added.